docs/resource/host.md

```diff
@@ -31,6 +31,19 @@
 | `x86` | 32-bit x86 |
 <!-- endsemconv -->
 
+**type:** `host.cpu`
+
+<!-- semconv host.cpu -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `host.cpu.vendor_id` | string | Processor manufacturer identifier. A maximum 12-character string. | `GenuineIntel` | Opt-In |
+| `host.cpu.family` | int | Numeric value specifying the family or generation of the CPU. | `6` | Opt-In |
+| `host.cpu.model` | int | Model identifier. It provides more granular information about the CPU, distinguishing it from other CPUs within the same family. | `6` | Opt-In |
+| `host.cpu.model_name` | string | Model designation of the processor. | `11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz` | Opt-In |
+| `host.cpu.stepping` | int | Stepping or core revisions. | `1` | Opt-In |
+| `host.cpu.cache_size` | int | The amount of level 2 memory cache available to the processor (in Bytes). | `12288000` | Opt-In |
```

Comment:
I am still not familiar enough with the 'rules' we use for conventions naming, so I have a couple questions:

1. Should we have `host.cpu.model.id` and `host.cpu.model.name` instead of the current wording? Since they both seem to refer to properties of the model. `stepping` could also be part of the model, at least semantically.
2. Similarly, do we expect to ever report properties about other caches (other than L2) or properties other than the size? Should that make us change into... `cache.size`? `cache.l2.size`? It's still unclear to me what level of specificity we need in the naming (it looks like I tend to want to be more specific than most, based on the `host.ip` discussion).

Reply:
yeap, I had the same though about grouping those. I think it's more future proof to follow what you mention. I tried to apply this with [356e3f8](https://github.com/open-telemetry/semantic-conventions/pull/209/commits/356e3f8a293cad107ac1ba3c47df0d531562d2dd).

Reply:
Agree with the `host.cpu.model.*` as a namespace.
But what about `host.cpu.vendor.id`? Do we expect other attributes under `host.cpu.vendor.*`? If not maybe it's better to leave it as `host.cpu.vendor_id`?

Reply:
For `cache` maybe in the future there will be need to store the cache's blocks or cache model name. Also other levels of cache might be available as well `L2`, `L3` etc.

For vendor ID I think it's most specific like this and we might need the Vendor info directly in the future. For example there are `AMDisbetter!` and `AuthenticAMD` as possible vendor_id values but both come from `AMD`, the only difference it's their historical date.

In any case I don't have a strong preference here, so I would be ok to change those if we don't see a lot of value.

Reply:
Marking as solved since the current changes are already approved.